frontends/web/.eslintrc.json

```diff
@@ -3,7 +3,8 @@
   "rules": {
     "jsx-a11y/anchor-is-valid": 0,
     "jsx-a11y/alt-text" : 0,
-    "quotes": ["error", "single"]
+    "quotes": ["error", "single"],
+    "indent": ["error", 2, { "SwitchCase": 1 }]
```

Comment:
could you also add jsx quotes? https://eslint.org/docs/rules/jsx-quotes#prefer-double

Reply:
I'd also like enforce consistent spacing inside braces (object-curly-spacing)
@benma @x1ddos @BrianCraig WDYT?

could we enforce `"object-curly-spacing": ["error", "always"],` 
https://eslint.org/docs/rules/object-curly-spacing#always